### DISCUSSION TITLE
I´ve tried with CachyOS and it works flawlessly

### DISCUSSION THREAD

**Discussion by DavidPerezIngeniero** [yuezk/GlobalProtect-openconnect#419] [Show and tell]
In that idstribution I cannot use the official GlobalProtect.
I'm very grateful.

**Comment by yuezk** [comment]
Thanks for your feedback.

**Comment by DavidPerezIngeniero** [comment]
The only downside with the official client, is that if I disconnect and reconnect shortly, I have to authenticate another time.
